printer/PrintHandler.js

```diff
@@ -29,6 +29,21 @@ const params = {
   WaitTimeSeconds: 0,
 };
 
+function deleteFile(fileNo) {
+  const paramers = {
+    Bucket: PRINTING_BUCKET_NAME,
+    Key: `folder/${fileNo}.pdf`,
+  };
+
+  s3.deleteObject(paramers, function (err, data) {
+    if (err)
+      console.log(err);
```

Comment:
use the logger file rather than this. We have logger file just for this import it from util folder. const logger = require('../util/logger.js');